components/scream/src/physics/rrtmgp/tests/rrtmgp_test_utils.cpp

```diff
@@ -40,15 +40,20 @@ namespace rrtmgpTest {
     void dummy_atmos(
             std::string inputfile, 
             int ncol, real2d &p_lay, real2d &t_lay,
-            real2d &sfc_alb_dir, real2d &sfc_alb_dif, real1d &mu0,
+            real1d &sfc_alb_dir_vis, real1d &sfc_alb_dir_nir,
+            real1d &sfc_alb_dif_vis, real1d &sfc_alb_dif_nir,
+            real1d &mu0,
             real2d &lwp, real2d &iwp, real2d &rel, real2d &rei, real2d &cld) {
 
         // Setup boundary conditions, solar zenith angle, etc
         // NOTE: this stuff would come from the model in a real run
 
         // Ocean-ish values for surface albedos, just for example
-        memset(sfc_alb_dir , 0.06_wp );
-        memset(sfc_alb_dif , 0.06_wp );
+        memset(sfc_alb_dir_vis , 0.06_wp );
```

Comment:
Hmm it’s a bit of a bummer that the rrtmgp_standalone test requires us to change this here, as this would prevent us from testing with band-varying surface albedos in the future, and also causes this test to diverge a little more from what was in the RRTMGP repo. I don’t really see an easy way around this though, and I don’t see any plans to change how the model handles surface albedos, so this is probably fine.